wp-resources/plugins/sil-dictionary-webonary/webonary/Webonary_SemanticDomains.php

```diff
@@ -379,10 +379,36 @@ public static function GetTranslatedList(string $lang_code = '', bool $include_d
 
 	public static function GetDropdown(string $lang_code = ''): string
 	{
-		// get the list of domains
-		$domains = self::GetTranslatedList($lang_code, false);
-		if (empty($domains))
-			return '';
+		// check for cached list
```

Comment:
@hanatgit, I added code to cache the filtered list.

Reply:
Great idea, @phillip-hopper ! Your code LGTM.